packages/components/src/ProgressBar/ProgressBar.test.tsx

```diff
@@ -16,4 +16,25 @@ describe("with props", () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  describe("stepped variation", () => {
+    it("renders correctly", () => {
```

Comment:
I'm not sure if this is too generic; I could separate these out into separate assertions if that's preferred.

Reply:
yeah, should separate them out.  Basically, the description should tell you what it's testing. That helps in understanding the test better. And also help you write them haha.

Reply:
Applied in [`ecc624d`](https://github.com/GetJobber/atlantis/pull/1653/commits/ecc624d5423292ffdb6c1dc9346b313145e15492)!